R/sequences.R

```diff
@@ -10,7 +10,10 @@
 #' @inheritParams valuetype
 #' @param case_insensitive ignore case when matching, if \code{TRUE}
 #' @param count_min minimum frequency of sequences
-#' @param nested collect nested sequences
+#' @param len_max maxium length of sequences
+#' @param nested if true, collect nested sequences
```

Comment:
Can you elaborate here? Not clear to me what nested sequences are or what this argument type is (logical I am guessing)